tests/quality/test_quality.py

```diff
@@ -0,0 +1,123 @@
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import cross_val_score
+
+from rdt import HyperTransformer
+from rdt.transformers import NumericalTransformer, get_transformers_by_type
+from tests.quality.utils import download_single_table_dataset
+
+THRESHOLD = 0.4
+
+TYPE_TO_DTYPE = {
+    'numerical': ['number'],
+    'float': ['float'],
+    'int': ['int'],
+    'categorical': ['object', 'category'],
+    'datetime': ['datetime'],
+    'boolean': ['bool']
+}
+
+
+def format_array(array):
+    if not isinstance(array, np.ndarray):
+        array = array.to_numpy()
+
+    if len(array.shape) == 1:
+        array = array.reshape(-1, 1)
+    return array
+
+
+def get_regression_score(features, target):
+    model = LinearRegression()
+    scores = cross_val_score(model, features, target)
```

Comment:
What metric is being used here? If `cross_val_score` uses the model `.score` method it may end up using `mse`, which is not what we want.

Reply:
It uses the default for the estimator which is coefficient of determination for `LinearRegression`. 
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html
https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.cross_val_score.html

Reply:
Oh, cool! I thought the default for LR was mse! Good then